components/dashboard/src/menu/Menu.tsx

```diff
@@ -52,64 +47,49 @@ export default function Menu() {
         ]).then((setters) => setters.forEach((s) => s()));
     }, [setCurrency, setIsChargebeeCustomer, setIsStudent]);
 
-    const leftMenu: Entry[] = [
-        {
-            title: "Workspaces",
-            link: "/workspaces",
-            alternatives: ["/"],
-        },
-        {
-            title: "Projects",
-            link: `/projects`,
-            alternatives: [] as string[],
-        },
-    ];
-
-    const adminMenu: Entry = {
-        title: "Admin",
-        link: "/admin",
-        alternatives: [
-            ...getAdminTabs().reduce(
-                (prevEntry, currEntry) =>
-                    currEntry.alternatives
-                        ? [...prevEntry, ...currEntry.alternatives, currEntry.link]
-                        : [...prevEntry, currEntry.link],
-                [] as string[],
-            ),
-        ],
-    };
+    const adminMenu: Entry = useMemo(
+        () => ({
+            title: "Admin",
+            link: "/admin",
+            alternatives: [
+                ...getAdminTabs().reduce(
+                    (prevEntry, currEntry) =>
+                        currEntry.alternatives
+                            ? [...prevEntry, ...currEntry.alternatives, currEntry.link]
+                            : [...prevEntry, currEntry.link],
+                    [] as string[],
+                ),
+            ],
+        }),
+        [],
+    );
 
-    const handleFeedbackFormClick = () => {
+    const handleFeedbackFormClick = useCallback(() => {
         setFeedbackFormVisible(true);
-    };
+    }, []);
 
-    const onFeedbackFormClose = () => {
+    const onFeedbackFormClose = useCallback(() => {
         setFeedbackFormVisible(false);
-    };
+    }, []);
 
     return (
         <>
-            <header className="app-container flex flex-col pt-4 space-y-4" data-analytics='{"button_type":"menu"}'>
-                <div className="flex h-10 mb-3">
-                    <div className="flex justify-between items-center pr-3">
-                        <Link to="/" className="pr-3 w-10">
+            <header className="app-container flex flex-col pt-4" data-analytics='{"button_type":"menu"}'>
+                <div className="flex justify-between h-10 mb-3 w-full">
```

Comment:
**thought**: The org menu seems to push the user menu icon beyond the viewport in some cases. We could introduce a custom max width instead of the `max-w-xs` used on the org menu but it's should be fine for now as it should be an edge case.

| Example A | Example B |
|-|-|
| <img width="1080" alt="Frame 1269-a" src="https://user-images.githubusercontent.com/120486/222374900-6ff92b2b-c24d-4ca8-a112-09a89b64dfc8.png"> | <img width="1080" alt="Frame 1269-b" src="https://user-images.githubusercontent.com/120486/222374905-5feec7e2-4996-407a-9688-09f0d48b7aa9.png"> |